05/app.js

```diff
@@ -9,12 +9,46 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const paragraphElements = document.querySelectorAll('p')
+const anchorElements = document.querySelectorAll('a')
+
+const paragraphArray = [...paragraphElements]
+
+paragraphArray.forEach(function(item){
+    const paragraphID = item.dataset.id
+    stats.paragraphs[paragraphID] = 0
+})
+
+const linkClickCounter = function(link){
+    const linkElement = link.getAttribute('href')
+    if(linkElement === '/dolor.html'){
```

Comment:
Nie chodzi o to, aby sprawdzać każdy możliwy link - jeśli byłoby ich na stronie 100 to byłoby to bardzo uciążliwe.
Spróbuj to zrobić bardziej elastycznie - tak jak  w przypadku paragrafu.